extensions/vscode/vscode.wake

```diff
@@ -62,9 +62,16 @@ export def vscode _: Result Path Error =
     require Pass packageJSON =
         setVersion (replace `~` "-" release) "{here}/package.json"
 
+    require Pass npm =
+        whichInEnvPath "npm"
+        | getOrFail "Unable to find npm in $PATH".makeError
+
+    def emsdk = dirname npm
```

Comment:
Maybe name this `emsdkDir` given that `emsdk` is also the name of the executable provided by emscripten, just to avoid potential confunsion?